src/workingtitle-vcockpits-instruments-navsystems/html_ui/WorkingTitle/Pages/VCockpit/Instruments/NavSystems/Shared/WT_MapElement.js

```diff
@@ -0,0 +1,384 @@
+class WT_MapElement extends MapInstrumentElement {
```

Comment:
I believe we decided when we set up our "shadow" filesystem that entirely new files that went into /WorkingTitle/Pages/ didn't need to have a prefix on them.   It's important for class names in code because everything gets glommed into one namespace, but since we load the js with explicit path names it's a bit redundant  having /WorkingTitle/Pages/WT_something.js.

Reply:
For classes, I think the cases where we have new ones we've not used an underscore separating the prefix (eg, WTDataStore).   But I think @cwburnett  may have said that he does?   I'm not sure we have a consistent style on this, come to thing of it.

Reply:
resolved with a8b9064